db/seeds.rb

```diff
@@ -63,6 +63,7 @@ def fetch!
   end
 end
 
+# TODO: how to best retrieve all the tracks
```

Comment:
The seeds script is now broken. How should I fix it?

Reply:
I'll do that now.